Dockerfile

```diff
@@ -1,62 +1,49 @@
 # To run: docker run -v /path/to/wsgi.py:/var/www/pidgin/wsgi.py --name=pidgin -p 81:80 pidgin
 # To check running container: docker exec -it pidgin /bin/bash
 
-FROM ubuntu:16.04
 
-ENV DEBIAN_FRONTEND=noninteractive
+FROM tiangolo/uwsgi-nginx:python3.6-alpine3.7
+
 
 ENV appname=pidgin
 
-RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends \
-    build-essential \
-    curl \
-    git \
-    # dependency for cryptography
-    libffi-dev \
-    # dependency for pyscopg2 - which is dependency for sqlalchemy postgres engine
-    libpq-dev \
-    # dependency for cryptography
-    libssl-dev \
-    libxml2-dev \
-    libxslt1-dev \
-    nginx \
-    python3 \
-    python3-dev \
-    python3-pip \
-    python3-setuptools \
-    sudo \
-    vim
-
-RUN ln -s /usr/bin/python3 /usr/local/bin/python \
-    && python -m pip install --upgrade pip \
-    && python -m pip install --upgrade setuptools \
-    && python -m pip install uwsgi \
-    && mkdir /var/www/$appname \
-    && mkdir -p /var/www/.cache/Python-Eggs/ \
-    && chown www-data -R /var/www/.cache/Python-Eggs/ \
-    && mkdir /run/nginx/
+# number of uwsgi worker processes
+ENV UWSGI_CHEAPER 2
+
+# ensure www-data user exists
+RUN addgroup -g 82 -S www-data \
+    && adduser -u 82 -D -S -G www-data www-data
```

Comment:
I'm pretty sure alpine nginx already creates an `nginx` user - why don't we change `nginx.conf` to use that?